Resources/config/services.xml

```diff
@@ -8,6 +8,8 @@
             <tag name="kernel.event_subscriber"/>
         </service>
 
+        <service id="httplug.collector.plugin_journal" class="Http\HttplugBundle\Collector\PluginJournal" public="false" />
```

Comment:
I would be happier if we had it completely separated. I don't think it makes the code cleaner.